### ISSUE TITLE
Latest release (3.0.4) crashes on macOS 10.13 High Sierra

### ISSUE THREAD

**Issue by ticky** [buildkite/terminal#47]
```
failed MSpanList_Insert 0x535000 0x17258359846e 0x0
fatal error: MSpanList_Insert

runtime stack:
runtime.throw(0x4ac06b)
	/usr/src/go/src/runtime/panic.go:491 +0xad fp=0x7ffeefbff1c0 sp=0x7ffeefbff190
runtime.MSpanList_Insert(0x4d2648, 0x535000)
	/usr/src/go/src/runtime/mheap.c:692 +0x8f fp=0x7ffeefbff1e8 sp=0x7ffeefbff1c0
MHeap_FreeSpanLocked(0x4cf240, 0x535000, 0x100)
	/usr/src/go/src/runtime/mheap.c:583 +0x163 fp=0x7ffeefbff228 sp=0x7ffeefbff1e8
MHeap_Grow(0x4cf240, 0x8, 0x0)
	/usr/src/go/src/runtime/mheap.c:420 +0x1a8 fp=0x7ffeefbff268 sp=0x7ffeefbff228
MHeap_AllocSpanLocked(0x4cf240, 0x1, 0x0)
	/usr/src/go/src/runtime/mheap.c:298 +0x365 fp=0x7ffeefbff2a8 sp=0x7ffeefbff268
mheap_alloc(0x4cf240, 0x1, 0x12, 0x0)
	/usr/src/go/src/runtime/mheap.c:190 +0x121 fp=0x7ffeefbff2d0 sp=0x7ffeefbff2a8
runtime.MHeap_Alloc(0x4cf240, 0x1, 0x10000000012, 0xfb39)
	/usr/src/go/src/runtime/mheap.c:240 +0x66 fp=0x7ffeefbff308 sp=0x7ffeefbff2d0
MCentral_Grow(0x4d6fb8, 0x0)
	/usr/src/go/src/runtime/mcentral.c:197 +0x8b fp=0x7ffeefbff370 sp=0x7ffeefbff308
runtime.MCentral_CacheSpan(0x4d6fb8, 0x0)
	/usr/src/go/src/runtime/mcentral.c:85 +0x167 fp=0x7ffeefbff3a8 sp=0x7ffeefbff370
runtime.MCache_Refill(0x531000, 0x12, 0x0)
	/usr/src/go/src/runtime/mcache.c:90 +0xa0 fp=0x7ffeefbff3d0 sp=0x7ffeefbff3a8
runtime.mcacheRefill_m()
	/usr/src/go/src/runtime/malloc.c:368 +0x57 fp=0x7ffeefbff3f0 sp=0x7ffeefbff3d0
runtime.onM(0x3c5130)
	/usr/src/go/src/runtime/asm_amd64.s:273 +0x9a fp=0x7ffeefbff3f8 sp=0x7ffeefbff3f0
runtime.mallocgc(0x120, 0x31bb80, 0x0, 0x0)
	/usr/src/go/src/runtime/malloc.go:178 +0x849 fp=0x7ffeefbff4a8 sp=0x7ffeefbff3f8
runtime.newobject(0x31bb80, 0x531000)
	/usr/src/go/src/runtime/malloc.go:353 +0x49 fp=0x7ffeefbff4d0 sp=0x7ffeefbff4a8
runtime.newG(0x28faa)
	/usr/src/go/src/runtime/proc.go:233 +0x2a fp=0x7ffeefbff4e8 sp=0x7ffeefbff4d0
allocg(0x4bfc20)
	/usr/src/go/src/runtime/proc.c:925 +0x1f fp=0x7ffeefbff4f8 sp=0x7ffeefbff4e8
runtime.malg(0x8000, 0x4bfde0)
	/usr/src/go/src/runtime/proc.c:2106 +0x1f fp=0x7ffeefbff528 sp=0x7ffeefbff4f8
runtime.mpreinit(0x4c0400)
	/usr/src/go/src/runtime/os_darwin.c:137 +0x27 fp=0x7ffeefbff540 sp=0x7ffeefbff528
mcommoninit(0x4c0400)
	/usr/src/go/src/runtime/proc.c:201 +0xc9 fp=0x7ffeefbff568 sp=0x7ffeefbff540
runtime.schedinit()
	/usr/src/go/src/runtime/proc.c:138 +0x55 fp=0x7ffeefbff590 sp=0x7ffeefbff568
runtime.rt0_go(0x7ffeefbff5c8, 0x1, 0x7ffeefbff5c8, 0x0, 0x0, 0x1, 0x7ffeefbff7e0, 0x0, 0x7ffeefbff81b, 0x7ffeefbff84c, ...)
	/usr/src/go/src/runtime/asm_amd64.s:95 +0x116 fp=0x7ffeefbff598 sp=0x7ffeefbff590
```

This issue appears to be fixed by some combination of Go 1.9 and building on High Sierra.

**Comment by chezhe** [comment]
Same problem, when I open my terminal. It should be a promblem of Go.

**Comment by ticky** [comment]
We’ve now released [v3.0.5](https://github.com/buildkite/terminal/releases/latest), which fixes this issue.
